Makefile

```diff
@@ -90,6 +88,9 @@ golangci-lint: ## Download golangci-lint locally if necessary.
 gcov2lcov: ## Download gcov2lcov locally if necessary.
 	$(call go-install-tool,$(GCOV2LCOV),github.com/jandelgado/gcov2lcov@$(GCOV2LCOV_VERSION))
 
+setup-envtest: ## Download setup-envtest locally if necessary
+	$(call go-install-tool,$(SETUP_ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
```

Comment:
nit: do we want to stick to a particular envtest version or fine to keep tracking `latest`?

Reply:
I think it's better to pin to a particular release. The only thing is that the version would be called `release-0.19`, instead of `0.19.0`. We can't use the semver tag as setup-envtest is a go submodule, but I guess this is just a cosmetic issue

Reply:
@vasrem I've updated the PR. Please check again